client/cl_main.lua

```diff
@@ -134,16 +146,16 @@ local function RotationToDirection(rotation)
 end
 
 function Camera(ped)
+    DoingCamera = true
     local cameraRotation = GetGameplayCamRot()
 	local cameraCoord = GetGameplayCamCoord()
 	local direction = RotationToDirection(cameraRotation)
 	local destination =
 	{
-		x = cameraCoord.x + direction.x * 10.0,
-		y = cameraCoord.y + direction.y * 10.0,
-		z = cameraCoord.z + direction.z * 10.0
+		x = cameraCoord.x + direction.x * 1000.0,
+		y = cameraCoord.y + direction.y * 1000.0,
+		z = cameraCoord.z + direction.z * 1000.0
```

Comment:
you dont need it to be 1000, you can get away with 20 or something once you reduce the distance above

Reply:
my bad, i forgot to revert these. testing